pom.xml

```diff
@@ -51,31 +60,57 @@
             <artifactId>spring-boot-starter-social-twitter</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.springframework.social</groupId>
+            <artifactId>spring-social-core</artifactId>
+            <version>1.1.4.RELEASE</version>
```

Comment:
Try to move into maven properties tag like : <swagger-ui.version>2.6.1</swagger-ui.version>